features/extension-mgt/org.wso2.carbon.identity.extension.mgt.feature/resources/extensions/connections/twofactor-custom-authentication/info.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "twofactor-custom-authentication",
+  "name": "2FA Authentication",
+  "description": "Only verify users in a second or later step of the login flow.",
+  "image": "",
+  "category": "DEFAULT",
+  "displayOrder": -1,
+  "tags": [ "CUSTOM" ],
```

Comment:
Add 2FA tag

Reply:
This is updated with 7d2336487ac11a54b528c7c248af61bb678b98e8